resources/operator.yaml

```diff
@@ -15,6 +15,17 @@ spec:
     spec:
       serviceAccountName: benchmark-operator
       containers:
+        - name: ansible
+          command:
+          - /usr/local/bin/ao-logs
+          - /tmp/ansible-operator/runner
+          - stdout
+          image: quay.io/benchmark-operator/benchmark-operator:master
+          imagePullPolicy: "Always"
+          volumeMounts:
+          - mountPath: /tmp/ansible-operator/runner
```

Comment:
Not sure whats happening but https://github.com/cloud-bulldozer/ripsaw/blob/master/tests/test_uperf.sh#L24 seems to be causing an issue. were youa ble to run the test locally?